Automattic/wc-calypso-bridge
# Launch Your Store: Disable Core's feature

### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/team-ghidorah/issues/298

Disable Launch Your Store feature that will be shipped in Core. Woo Express already has a similar mechanism and until the two methods are reconciled, disable the core version by forcing the feature flag to false.






### How to test the changes in this Pull Request:




1. Use your WooExpress testing setup
2. Using the CLI make sure you have the latest WooCommerce
```
wp plugin delete woocommerce
wp plugin install https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip --activate
```
3. Make sure you have the Beta Tester
```
wp plugin install https://github.com/woocommerce/woocommerce/releases/download/wc-beta-tester-2.2.5/woocommerce-beta-tester.zip --activate
```
4. Turn on the `launch-your-store` feature flag Tools > WCA Test Helper > Features
5. Make sure this code is present using your atomic-watch script or whichever you use.
6. In the console, confirm the flag is turn off by ensuring `wcAdminFeatures['launch-your-store']` returns undefined.



### Other information:




### FOR PR REVIEWER ONLY:
